nox/manifest.py

```diff
@@ -164,6 +164,37 @@ def make_session(self, name, func):
     def next(self):
         return self.__next__()
 
+    def notify(self, session):
```

Comment:
How does this work with parametrized sessions?

Reply:
It would have to be `notify('unit_tests', python_version='3.6')`. I agree that would be nice but assert it can be separate. (Both forms can work.)

Reply:
that's fine

Reply:
Do you want to add this in a follow-up PR? If so, LGTM.